math/divisor.cpp

```diff
@@ -0,0 +1,97 @@
+/**
+ * @file
+ * @brief Algorithms to find divisor of all natural numbers.
+ * @author [Tee Chia Suen](https://github.com/ChiaSuen)
+ */
+
+#include <cassert>  /// for assert
+#include <iostream>
+#include <vector>  //for declaring undefined size array
```

Comment:
```suggestion
#include <cassert>  /// for assert
#include <iostream> /// for IO operations
#include <vector>  /// for declaring undefined size arrays
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inelegant